src/include/processor/operator/mask.h

```diff
@@ -58,6 +58,9 @@ class MaskCollection {
     // Note: blindly update mask does not parallelize well, so we minimize write by first checking
     // if the mask is set to true (mask value is equal to the expected currentMaskValue) or not.
     void incrementMaskValue(common::offset_t offset, uint8_t currentMaskValue) {
+        if (offset >= maskData->getSize()) { // Handle uncommitted node offsets.
```

Comment:
Can u remove these lines of code?
https://github.com/kuzudb/kuzu/blob/11b093bda0632b201ae43de07b12fbf074df67dc/src/processor/operator/semi_masker.cpp#L37-L39
I think I only protected it against case of SingleTableSemiMasker. I would also mark `unlikely` here.